src/translatador.cpp

```diff
@@ -0,0 +1,385 @@
+﻿#include <translatador.h>
+
+#include <marian.h>
+#include <common/options.h>
+#include <data/types.h>
+#include <translator/annotation.h>
+#include <translator/beam_search.h>
+#include <translator/definitions.h>
+#include <translator/logging.h>
+#include <translator/text_processor.h>
+#include <variant>
+
+namespace bergamot = marian::bergamot;
+
+static std::mutex init_mutex;
+static bool initialized;
+
+static thread_local std::string last_error;
+
+void initialize() {
+    std::lock_guard guard(init_mutex);
```

Comment:
This function should probably be static, as it doesn't need to exist outside of this translation unit